v2/googlecloud-to-googlecloud/README_Bigtable_Change_Streams_to_PubSub.md

```diff
@@ -0,0 +1,234 @@
+Cloud Bigtable change streams to PubSub Template
+---
+Streaming pipeline. Streams Bigtable data change records and writes them into PubSub using Dataflow Runner V2.
+
+:memo: This is a Google-provided template! Please
+check [Provided templates documentation](https://cloud.google.com/dataflow/docs/guides/templates/provided-templates)
+on how to use it without having to build from sources using [Create job from template](https://console.cloud.google.com/dataflow/createjob?template=Bigtable_Change_Streams_to_PubSub).
+
+
+:bulb: This is a generated documentation based
+on [Metadata Annotations](https://github.com/GoogleCloudPlatform/DataflowTemplates#metadata-annotations)
+. Do not change this file directly.
+
+## Parameters
+
+### Required Parameters
+
+* **pubSubTopic** (The output Pub/Sub topic): The Pub/Sub topic to publish changelog entry messages.
+* **bigtableChangeStreamAppProfile** (Cloud Bigtable application profile ID): The application profile is used to distinguish workload in Cloud Bigtable.
+* **bigtableReadInstanceId** (Source Bigtable Instance ID): The ID of the Cloud Bigtable instance that contains the table.
+* **bigtableReadTableId** (Source Cloud Bigtable table ID): The Cloud Bigtable table to read from.
+
+### Optional Parameters
+
+* **messageEncoding** (The encoding of the message written into PubSub): The format of the message to be written into PubSub. Allowed formats are BINARY and JSON Text. Default value is JSON.
```

Comment:
> Allowed formats are BINARY and JSON Text

Let's remove Text from here, it really confuses the reader whether the value to be specified is "JSON" or "JSON Text". I think JSON implies text anyway